src/transformers/models/wav2vec2/configuration_wav2vec2.py

```diff
@@ -329,3 +331,7 @@ def __init__(
         self.tdnn_kernel = list(tdnn_kernel)
         self.tdnn_dilation = list(tdnn_dilation)
         self.xvector_output_dim = xvector_output_dim
+
+        @property
+        def inputs_to_logits_ratio(self):
```

Comment:
Can quickly change name if you have better ideas, but think that's pretty clear cc @Narsil